api/src/pcapi/serialization/decorator.py

```diff
@@ -166,7 +175,7 @@ def sync_validate(*args: Any, **kwargs: Any) -> Response:
                         400,
                     )
             if query_in_kwargs:
-                content = request.args.to_dict(flat=False) if flatten else query_params
+                content = transform_query_args_to_dict(query_params)
```

Comment:
Ceci devrait-être opt-in pour ne pas que toutes les requêtes avec query params soient impactées